src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -28,6 +28,8 @@ const Microsoft.CodeAnalysis.WellKnownMemberNames.TopLevelStatementsEntryPointTy
 Microsoft.CodeAnalysis.CompilationOptions.SyntaxTreeOptionsProvider.get -> Microsoft.CodeAnalysis.SyntaxTreeOptionsProvider
 Microsoft.CodeAnalysis.CompilationOptions.WithSyntaxTreeOptionsProvider(Microsoft.CodeAnalysis.SyntaxTreeOptionsProvider provider) -> Microsoft.CodeAnalysis.CompilationOptions
 Microsoft.CodeAnalysis.Operations.IPatternOperation.NarrowedType.get -> Microsoft.CodeAnalysis.ITypeSymbol
+static Microsoft.CodeAnalysis.SyntaxNodeExtensions.WithoutLeadingTrivia(this Microsoft.CodeAnalysis.SyntaxToken token) -> Microsoft.CodeAnalysis.SyntaxToken
+static Microsoft.CodeAnalysis.SyntaxNodeExtensions.WithoutTrailingTrivia(this Microsoft.CodeAnalysis.SyntaxToken token) -> Microsoft.CodeAnalysis.SyntaxToken
```

Comment:
InternalsVisibleTo didn't seem to be in place.

Reply:
i would do this in the IDE layer.

Reply:
can you moved these @jnm2 ?

Reply:
Done. Had no idea if you'd like it if I added an unused extension method, so I didn't.